src/lang/modifyAst.ts

```diff
@@ -443,30 +443,54 @@ export function loftSketches(
   }
 }
 
-export function addSweep(
-  node: Node<Program>,
-  profileDeclarator: VariableDeclarator,
-  pathDeclarator: VariableDeclarator
-): {
+export function addSweep({
+  node,
+  targetDeclarator,
+  trajectoryDeclarator,
+  sectional,
+  variableName,
+  insertIndex,
+}: {
+  node: Node<Program>
+  targetDeclarator: VariableDeclarator
+  trajectoryDeclarator: VariableDeclarator
+  sectional: boolean
+  variableName?: string
+  insertIndex?: number
+}): {
```

Comment:
Changing to an object of args to be more consistent with other commands, and taking the opportunity to use more consistent lingo with the rest of ts and engine api for target and trajectory